paypal/controllers/FrmPayPalLiteFieldsController.php

```
@@ -0,0 +1,40 @@
+
+		$is_paypal_order_field = FrmField::get_option( $field, 'is_paypal_order_field' );
+
+		return $is_paypal_order_field ? false : $show_normal_field_type;
+	}
+
+	/**
+	 * @param array $field The field settings.
+	 * @param array $display The display settings for the field.
+	 * @param array $values The values associated with the field.
+	 */
+	public static function add_paypal_order_field_note( $field, $display, $values ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$display'. (undefined)

(UnusedFormalParameter)